rules-tests/Php81/Rector/MethodCall/SpatieEnumMethodCallToEnumConstRector/Fixture/fully_qualified_class_name.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\MethodCall\SpatieEnumMethodCallToEnumConstRector\Fixture;
+
+use \Rector\Tests\Php81\Rector\MethodCall\SpatieEnumMethodCallToEnumConstRector;
+
+final class FullyQualifiedClassName
+{
+    public function run($value)
+    {
+        $archived = \Rector\Tests\Php81\Rector\MethodCall\SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::archived();
+        $published = SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::published();
+
+        $archivedValue = \Rector\Tests\Php81\Rector\MethodCall\SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::archived()->getValue();
+        $archivedName = \Rector\Tests\Php81\Rector\MethodCall\SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::archived()->getName();
+
+        $draftValue = SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::draft()->value;
+        $draftName = SpatieEnumMethodCallToEnumConstRector\Source\StatusEnum::draft()->name;
```

Comment:
please extract to 2 or 3 kind of fixtures to easier to patch when there is a bug in the future.